app/assets/stylesheets/forever_style_guide/globals/_effects.scss

```diff
@@ -18,6 +18,12 @@
   transition: $transition-properties;
 }
 
+@mixin transform($transform-properties...) {
```

Comment:
My only hesitation would be that changing the default supported browsers would require including a file at the root of the directory and I'm not sure if a rails engine can do that.  If it can, or if we don't ever want to change the defaults (they're pretty good currently), then it makes sense to add it to the style guide.